### PR TITLE
Add Persian Percentage Calculation + Fixing some important bugs :P

### PR DESCRIPTION
**Pull request by omid**

### CODE REVIEW

File: src/persian_content/mod.rs [hunk 1]
```diff
@@ -7,15 +7,42 @@ lazy_static! {
...
     /// Checks if a text has at least a Persian char in it.
     fn has_persian_char(&self) -> bool {
         HAS_PERSIAN_CHAR.is_match(self.as_ref())
     }
 
     /// Checks if a text is in Persian.
     fn is_persian_str(&self) -> bool {
-        PERSIAN_STR.is_match(self.as_ref())
+        // First remove the non-alphabetic chars
+        let str = self
```
Comment (robatipoor) [inline src/persian_content/mod.rs:19]: This code has both congestion and performance issues I think we'd better fix Regex

Comment (omid) [reply src/persian_content/mod.rs:19]: Any solution?

Comment (robatipoor) [reply src/persian_content/mod.rs:19]: `^[\u0600-\u06FF\s.,?!-_]+$`

Comment (omid) [reply src/persian_content/mod.rs:19]: So what about emojis? What about allllllll other unicode chars that are not alphabetic?

Comment (robatipoor) [reply src/persian_content/mod.rs:19]: `^[\u0600-\u06FF]|[[:punct:]]+$`

Comment (robatipoor) [reply src/persian_content/mod.rs:19]: about emoji I have no idea

Comment (omid) [reply src/persian_content/mod.rs:19]: There are about 12350 non-alphabetic chars in unicode, ref: https://www.compart.com/en/unicode/category

Comment (omid) [reply src/persian_content/mod.rs:19]: And you know, u0600-u06FF is not correct! :D

Comment (robatipoor) [reply src/persian_content/mod.rs:19]: Okay, so we'll merge this code, thanks

**Pull request merged**